### DISCUSSION TITLE
Default Lazer progress bar should have its text anchored right-side up like timing bar icons are

### DISCUSSION THREAD

**Discussion by CheeseManFuu** [ppy/osu#14947] [Ideas] [answered]
Super minor thing that I couldn't help but notice when playing with the UI Editor was that the progress bar's times and percentage were affected by transformations whereas the timing bar had its truck and bike icons scale and be unaffected by rotation and flipping. To have the progress bar on the top (and still move left-to-right) means flipping it vertically, which consequentially has the numbers also flipped vertically, and difficult to read. Simiarly, if you want the progress bar to be shorter or taller but the same width, to keep it out of the way of gameplay as possible but still visible, or less wide or wider but the same height, such as for people who use 4:3 or 21:9 resolutions, the text is squished or stretched. 

When it comes to stretching the element taller or wider, the text should remain the size it would be at the correct proportions, just be anchored to their respective corners.

A couple example screenshots and accompanying mock-ups of the fix below:

![osu_2021-10-03_18-01-48](https://user-images.githubusercontent.com/7471405/135775306-4cb8a831-cb80-4c53-83f9-654e7d781c2a.jpg)
Normal on the left
![osu_2021-10-03_18-01-48edit](https://user-images.githubusercontent.com/7471405/135775314-140b7a23-e4a8-41bc-8e2e-e40c43904d70.jpg)
Fixed on the left
![osu_2021-10-03_18-02-49](https://user-images.githubusercontent.com/7471405/135775317-3ee52606-5332-41e4-9024-e7ede2862cad.jpg)
Normal on the top
![osu_2021-10-03_18-02-49edit](https://user-images.githubusercontent.com/7471405/135775318-ecc64b2a-118f-49e4-95fb-c8695271013d.jpg)
Fixed on the top

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/14953
